src/Kafka/CreateKafkaTopic/components/WizardCustomFooter.tsx

```diff
@@ -0,0 +1,118 @@
+import type React from "react";
+import {
+  WizardContextConsumer,
+  Button,
+  WizardFooter,
+} from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+export interface IWizardFooter {
+  isLoading: boolean;
+  onValidate: (value: () => void) => void;
+  topicNameValidated: "error" | "default";
+  closeWizard: () => void;
+  partitionValue: number;
+}
+export const WizardCustomFooter: React.FC<IWizardFooter> = ({
+  isLoading,
+  onValidate,
+  closeWizard,
+  partitionValue,
+}) => {
+  const { t } = useTranslation(["create-topic", "common"]);
+
+  return (
+    <WizardFooter>
+      <WizardContextConsumer>
+        {({ activeStep, onNext, onBack }) => {
+          if (activeStep.name == t("topic_name")) {
```

Comment:
It would be better to use unique id for checking active step instead visual name value.